Stylesheets/components/badge.css

```diff
@@ -0,0 +1,73 @@
+@import url(/Stylesheets/main.css);
+
+.bedge {
+  margin: 2rem;
+}
+.bg {
```

Comment:
bg seems like background rather than badge, use proper naming conventions

Reply:
yes i have changed it